src/shell-window-tracker.c

```diff
@@ -782,6 +792,40 @@ shell_window_tracker_get_app_from_pid (ShellWindowTracker *tracker,
 }
 
 static void
+on_child_exited (GPid      pid,
+                 gint      status,
+                 gpointer  unused_data)
+{
+  ShellWindowTracker *tracker;
+
+  tracker = shell_window_tracker_get_default ();
+
+  g_hash_table_remove (tracker->launched_pid_to_app, GINT_TO_POINTER((gint)pid));
+}
+
+void
+_shell_window_tracker_add_child_process_app (ShellWindowTracker *tracker,
+                                             GPid                pid,
+                                             ShellApp           *app)
+{
+  gpointer pid_ptr = GINT_TO_POINTER((int)pid);
+
+  if (g_hash_table_lookup (tracker->launched_pid_to_app,
+                           &pid_ptr))
```

Comment:
It strikes me that this should be `pid_ptr`, not its address (which is a `void **`) unless I'm missing something